internal/tiltfile/sys/sys.go

```diff
@@ -0,0 +1,53 @@
+package sys
+
+import (
+	"os"
+
+	"go.starlark.net/starlark"
+
+	"github.com/tilt-dev/tilt/internal/tiltfile/starkit"
+)
+
+// The starlark sys module.
+// Contains a subset of Python's sys module.
+// https://docs.python.org/3/library/sys.html
+type Extension struct {
+}
+
+func NewExtension() Extension {
+	return Extension{}
+}
+
+func (e Extension) OnStart(env *starkit.Environment) error {
+	err := env.AddValue("sys.argv", argv())
+	if err != nil {
+		return err
+	}
+
+	err = env.AddValue("sys.executable", executable())
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// List of commandline arguments that Tilt started with.
+func argv() starlark.Value {
+	values := []starlark.Value{}
+	for _, arg := range os.Args {
+		values = append(values, starlark.String(arg))
+	}
+
+	list := starlark.NewList(values)
+	list.Freeze()
+	return list
+}
+
+// Full path to the Tilt executable.
+func executable() starlark.Value {
```

Comment:
Hm, I guess we should think about moving the `config` vars (`main_dir`, `main_path`, `tilt_subcommand`) into `sys`?

(I don't know if it's just me, but I've always struggled with the distinction between `os` and `sys` in Python, and now we've got that but also throwing in `config` with similar stuff)

Reply:
eh, i wouldn't stress too much about it.

i think these made sense under `sys` because of the consistency with python. i think the other ones are less interesting, because they don't really have a python analog